src/pages/resultsView/pathwayMapper/PathwayMapperMessageBox.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
```

Comment:
It's probably better to move this component under `src/shared/lib/pathwayMapper`.